web/src/stories/tabs/base.stories.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react'
+
+import {
+  Tabs,
+  TabPanels,
+  Tab,
+  TabsProps,
+} from '../../components/tabs/index.tsx'
+
+export default {
+  title: 'Components/Tabs',
+  component: Tabs,
+}
+
+export const Base: React.FC<TabsProps> = props => (
+  <Tabs {...props}>
+    <Tab>Tab name 1</Tab>
+    <Tab>Tab name 2 </Tab>
+    <Tab>Tab name 3</Tab>
+    <TabPanels>
+      <div>Tab 1 content</div>
+      <div>Tab 2 content</div>
+      <div>Tab 3 content</div>
+    </TabPanels>
+  </Tabs>
+)
```

Comment:
Can we move to the proposed API?
This feels unnatural to use! 
```
<Tabs>
  <Tab name="Tab name 1">
    <div>My tab body content</div>
  </Tab>
  <Tab name="Tab name 2">
    <div>My tab body content 2</div>
  </Tab>
</Tabs>
```

This way it's clear what tab contains what content. It's how other design systems is exposing it! Also, one component less to import :)

Reply:
Agree, just pushed changes